core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala

```diff
@@ -1300,7 +1304,9 @@ class GroupCoordinator(val brokerId: Int,
     completeAndScheduleNextHeartbeatExpiration(group, member)
 
     val knownStaticMember = group.get(newMemberId)
-    group.updateMember(knownStaticMember, protocols, responseCallback)
+    val oldRebalanceTimeoutMs = knownStaticMember.rebalanceTimeoutMs
+    val oldSessionTimeoutMs = knownStaticMember.sessionTimeoutMs
+    group.updateMember(knownStaticMember, protocols, rebalanceTimeoutMs, sessionTimeoutMs, responseCallback)
```

Comment:
Could we add a test in `GroupCoordinatorTest` to exercise this path and the failed path as well?

Reply:
I can not mock the failed case in test

Reply:
@Stephan14 Could you add a unit test in `GroupCoordinatorTest`? If you have trouble with this, could you explain your issues? I can perhaps help you.

Reply:
I have add unit test about persistence failed after join group successfully in `GroupCoordinatorTest`

Reply:
@dajac Can you review it?